src/Data/GameMaster.cs

```diff
@@ -112,6 +114,16 @@ public static PokedexPokemon GetPokemon(uint pokemonId, uint formId = 0)
             return pkmnForm;
         }
 
+        public static void ReloadMasterFile()
+        {
+
+            var path = Path.Combine(
+                        Strings.BasePath,
+                        Path.Combine(Strings.DataFolder, MasterFileName)
+                    );
+            _instance = LoadInit<GameMaster>(path);
```

Comment:
Just to clean up the spacing/empty lines
```suggestion
            var path = Path.Combine(
                Strings.BasePath,
                Path.Combine(Strings.DataFolder, MasterFileName)
            );
            _instance = LoadInit<GameMaster>(path);
```